packages/trader/src/AppV2/Components/RiskManagementItem/__tests__/risk-management-item.spec.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import RiskManagementItem from '../risk-management-item';
+import useContractDetails from 'AppV2/Hooks/useContractDetails';
+import { CONTRACT_TYPES } from '@deriv/shared';
+
+jest.mock('@deriv/translations', () => ({
+    Localize: ({ i18n_default_text }: { i18n_default_text: string }) => <span>{i18n_default_text}</span>,
```

Comment:
Sorry, do we really need to mock `Localize` ?

Reply:
No, but it's not related to testing this component, so it makes sense to mock any components that shouldn't be tested during the testing of the main component.